module_generator/cookiecutter-template/{{cookiecutter.module_name}}/requirements.txt

```diff
@@ -0,0 +1 @@
+viam-sdk==<CURRENT VERSION>
```

Comment:
should we add here also python version limitations? Since we don't support before 3.8.

Reply:
`requirements.txt` is more for package dependencies, so from what I know, you can't specify a python version here. I think we'd need a `pyproject.toml` file for that.